docs/user_guide/HOWTO/IPA_Clients.rst

```diff
@@ -0,0 +1,86 @@
+.. _howto-ipa_clients:
+
+HOWTO Enroll Hosts into an IPA Domain
+=====================================
+
+Host should be able to join an IPA domain with a few catches. SIMP already uses
+the login stack that IPA uses, but it also actively manages the same resources
+that IPA provides automation for, like:
+
+#. autofs
```

Comment:
We only manage `sudoers` and `SELinux` by default and we don't manage the same parts of `SELinux` that IPA does so that shouldn't be an issue at all.

Also, we should add a ticket for hooking `sudoers` to IPA/LDAP or however that works internally.

Reply:
I think there is a ticket for looking into sudoers integration. I'll link it here. 

SELinux here refers what IPA calls 'SELinux User Maps', which is something different, I think. But again, we haven't looked into it.

I think I need to categorize these a little better.